Web/starter_project_vanguard/components/home/Landing_section/Landing.tsx

```diff
@@ -0,0 +1,46 @@
+import React from 'react'
+import { Buttons } from './Buttons'
+import Image from 'next/image'
+
+const Landing = () => {
+  return (
+    <div className="mt-20 mb-10">
+      <div className="grid w-full grid-cols-2 gap-10 p-10 pl-4 mt-8 place-content-center place-items-center">
+        {/* left part goes here */}
+        <div className="w-full ml-4">
+          <div>
+            {/* heading goes here */}
+            <h1 className="text-4xl font-extrabold sm:mb-7">
+              Africa To
+              <span className="text-blue-600"> Silicon Valley</span>
+            </h1>
+
+            {/* text goes here */}
+            <p className="mt-3 text-gray-700 whitespace-normal font-body md:text-lg md:font-semibold">
+              A2SV up-skills high-potential university students, connects them
+              with opportunities at top tech companies.
+            </p>
+
+            {/* buttons go here */}
```

Comment:
"buttons" does not describe what buttons these are supposed to be (especially for someone that has been reviewing different parts of the doc) so I would remind the naming improvement